org/qmckl.org

```diff
@@ -220,8 +220,8 @@ cppcheck --addon=cert --enable=all *.c &> cppcheck.out
    | =*_func.h=         | Public  | Function definitions         |
    | =*_private_type.h= | Private | Type definitions             |
    | =*_private_func.h= | Private | Function definitions         |
-   | =*fh_type.f90=     | Public  | Fortran type definitions     |
-   | =*fh_func.f90=     | Public  | Fortran function definitions |
```

Comment:
I changed it because in the description it said *_f_func.f90 and *_f_type.f90 but perhaps I can change the description and keep the table variable as is. Is this correct ?

Reply:
This has been restored and doc has been edited to be consistent. 7924600